internal/cli/atlas/backup/compliancepolicy/copyprotection/disable_test.go

```diff
@@ -43,13 +42,6 @@ func TestDisableBuilder(t *testing.T) {
 	)
 }
 
-func TestDisableOpts_InitStore(t *testing.T) {
-	opts := &DisableOpts{}
-
-	require.NoError(t, opts.initStore(context.TODO())())
-	assert.NotNil(t, opts.store)
-}
```

Comment:
we added a validation on the general store for the service and this now fails, deiced to remove since we never test this method

Reply:
same for all removals of this test in backup compliance